src/settings/App.tsx

```diff
@@ -47,13 +50,50 @@ export class App extends Component<{}, AppState> implements OC.Plugin<OC.Search.
 		sortOrder: 1
 	};
 
+	constructor(props) {
+		super(props);
+		// Enable the app if the user is member of a group with delegated admin rights
+		this.api.isGroupFoldersAdmin(OC.getCurrentUser().uid).then((isAdmin) =>  {
+			this.state.enabled = isAdmin;
```

Comment:
The idea is to enable/disable groupfolders based on this state, but it's not implemented yet.

Reply:
We could aso do this check in the backend and just return null for the settings section in case the settings shouldn't show up: https://github.com/nextcloud/groupfolders/blob/bff7bdd700b0c607d7a8cf95ff2ae750a29d08f6/lib/Settings/Admin.php#L44

Reply:
Good idea. I've implemented it